package.json

```diff
@@ -36,12 +36,12 @@
     "@ckeditor/ckeditor5-theme-lark": "40.1.0",
     "@ckeditor/ckeditor5-upload": "40.1.0",
     "eslint": "^7.32.0",
-    "eslint-config-ckeditor5": "^5.1.1",
+    "eslint-config-ckeditor5": "^5.1.2",
```

Comment:
It was a part of https://github.com/isaul32/ckeditor5-math/commit/fffb69be084f12e9e8bc694a473b7cdb0fe0fa8e but not https://github.com/bethinkpl/ckeditor5-math/commit/41f075f9d12215f00be10a973f10649a84cab174

Reply:
Without it, I was getting error about node 20 not being supported.